config.py

```diff
@@ -6,3 +6,4 @@
 JOKES_SOURCE_FILE = 'data/jokes.json'
 MAPQUEST_CONSUMER_KEY = '<MAPQUEST_CONSUMER_KEY>'
 TIME_ZONE_DB_API_KEY = '<TIME_ZONE_DB_API_KEY>'
+OPEN_WEATHER_MAP_ACCESS_TOKEN = '7af6de37fc4d3517129290b740663142'
```

Comment:
It's a trash account anyways, just proof that it works:)
On 8 Jun 2016 10:37 p.m., "Issa Jubril" <notifications@github.com> wrote:

> In config.py
> <https://github.com/swapagarwal/JARVIS-on-Messenger/pull/79#discussion_r66345071>
> :
>
> > @@ -6,3 +6,4 @@
> >  JOKES_SOURCE_FILE = 'data/jokes.json'
> >  MAPQUEST_CONSUMER_KEY = '<MAPQUEST_CONSUMER_KEY>'
> >  TIME_ZONE_DB_API_KEY = '<TIME_ZONE_DB_API_KEY>'
> > +OPEN_WEATHER_MAP_ACCESS_TOKEN = '7af6de37fc4d3517129290b740663142'
>
> I don't think having your access token in a config file is a good idea.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/swapagarwal/JARVIS-on-Messenger/pull/79/files/0edcea4a4bcb5d4adf0ed185e5488cf0d8a281c7#r66345071>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APup34l1vRYgX5ywbXmuiwsXdWs4_po6ks5qJzYtgaJpZM4IxLfd>
> .
>